src/components/plots/abstractPlot.ts

```diff
@@ -516,30 +516,17 @@ export module Plot {
     }
 
     /**
-     * Retrieves the closest PlotData for the specified dataset(s)
+     * Retrieves the closest PlotData across all datasets, where distance is defined to be
+     * the Euclidiean norm.
      *
-     * @param {Point} queryPoint The point to query from
-     * @param {number} withinValue Will only return plot data that is of a distance below withinValue
-     *                             (default = Infinity)
-     * @param {string | string[]} datasetKeys The dataset(s) to retrieve the plot data from.
-     *                                        (default = this.datasetOrder())
-     * @returns {PlotData} The retrieved PlotData.
+     * @param {Point} queryPoint The point to which dataset points should be compared
```

Comment:
Let's change the doc possibly to 

Retrieves PlotData with the lowest distance, where distance is defined to be the Euclidiean norm.